src/deadline/client/api/_loginout.py

```diff
@@ -46,16 +46,12 @@ def _login_deadline_cloud_monitor(
             # We don't hookup to stdin but do this to avoid issues on windows
             # See https://docs.python.org/3/library/subprocess.html#subprocess.STARTUPINFO.lpAttributeList
             p = subprocess.Popen(
-                args, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, stdin=subprocess.PIPE
+                args, stdout=subprocess.PIPE, stderr=subprocess.DEVNULL, stdin=subprocess.PIPE
             )
         else:
             p = subprocess.Popen(
-                args, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, stdin=subprocess.DEVNULL
+                args, stdout=subprocess.PIPE, stderr=subprocess.DEVNULL, stdin=subprocess.DEVNULL
```

Comment:
It won't let me comment outside of this context but the comment on line 76 was already incorrect as it refers to reading from stderr